segment_eval_inference.py

```diff
@@ -0,0 +1,187 @@
+"""Eval mAP@N metric from inference file."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from absl import app
+from absl import flags
+
+import mean_average_precision_calculator as map_calculator
+import numpy as np
+import tensorflow as tf
+
+flags.DEFINE_string(
+    "eval_data_pattern", "",
+    "File glob defining the evaluation dataset in tensorflow.SequenceExample "
+    "format. The SequenceExamples are expected to have an 'rgb' byte array "
+    "sequence feature as well as a 'labels' int64 context feature.")
+flags.DEFINE_string(
+    "label_cache", "",
```

Comment:
It seems kind of user unfriendly to have to manually manage this cache in order for the job to run quickly. I think it will be ok if we make sure it's included in the getting started documentation.

Reply:
Will update the doc in the next PR shortly.